packages/ui/src/working-groups/components/OpeningsList/Opening/OpeningListItem.tsx

```diff
@@ -33,7 +33,9 @@ export const OpeningListItem = ({ opening, past, onClick }: OpeningListItemProps
       <ToggleableItemInfo>
         <ToggleableItemInfoTop>
           <Subscription title={`ID: ${opening.id}`}>ID: {opening.runtimeId}</Subscription>
-          <Subscription>Duration: {relativeTime(opening.createdAtBlock.timestamp)}</Subscription>
+          {isInFuture(opening.expectedEnding) && (
+            <Subscription>Ends {relativeTime(opening.expectedEnding)}</Subscription>
```

Comment:
Looks good for decided proposals but "Ends" vs "[Ends in](https://github.com/Joystream/pioneer/pull/3829#discussion_r1054157981)".

![filled](https://user-images.githubusercontent.com/31551045/211154233-9e838222-2fba-4e23-a141-57140529d549.png)

Reply:
Done